.github/workflows/develop.yml

```diff
@@ -116,6 +116,14 @@ jobs:
       - name: Start Apache
         run: service apache2 start
 
+      - name: Install memcache extension
+        run: |
+          apt-get -y update
+          apt-get -y install php5-memcache memcache
+
+      - name: Start Apache
+        run: service apache2 restart
+
```

Comment:
It looks like starting apache is happening twice. I'd remove the one above the memcache installation.

Reply:
Trying the docker service approach.